webapp-django/crashstats/crashstats/views.py

```diff
@@ -1763,6 +1764,44 @@ def crontabber_state(request, default_context=None):
     return render(request, 'crashstats/crontabber_state.html', context)
 
 
+@utils.json_view
+def crontabber_status(request):
+    api = CrontabberState()
+
+    # start by assuming the status is OK which means no jobs are broken
+    context = {'status': 'OK'}
+
+    all = api.get()['state']
+    broken = [
+        name for name, state in all.items()
+        if state['error_count']
+    ]
+    blocked = [
+        name for name, state in all.items()
+        if set(broken) & set(state['depends_on'])
```

Comment:
What's wrong with ``state['depends_on'] in broken``? It seems easier to understand to me, and I don't think perf matter a lot here (though I would suspect it's actually faster to just use ``in`` instead of creating 2 new sets).